x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/rest/action/role/RestQueryRoleAction.java

```diff
@@ -32,6 +34,7 @@
 import static org.elasticsearch.rest.RestRequest.Method.POST;
 import static org.elasticsearch.xcontent.ConstructingObjectParser.optionalConstructorArg;
 
+@ServerlessScope(Scope.INTERNAL)
```

Comment:
As suggested by @n1v0lg, marking it as `INTERNAL` for now. I will make it public in a followup PR (including specification update).